docs/source/reference/kubernetes/kubernetes-getting-started.rst

```diff
@@ -316,32 +318,104 @@ For example, to set custom environment variables and use GPUDirect RDMA, you can
            pod_config:
              ...
 
+.. _kubernetes-using-volumes:
+
+Mounting NFS and other volumes
+------------------------------
+
+Using the ``pod_config`` field, you can attach `Kubernetes volumes <https://kubernetes.io/docs/concepts/storage/volumes/>`_ to your SkyPilot pods.
+
+This can be done directly from the task YAML on a per-task basis, or :ref:`globally for all tasks<kubernetes-custom-pod-config>` in :code:`~/.sky/config.yaml`.
```

Comment:
For the 2nd case, can we add another snippet tab-set down below? The syntax is slightly different, so readers that got redirected to above may get confused.

Reply:
Updated to use nested tab-set, wdyt? Avoiding making the docs too long